<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPUFrequencyControl</class>
 <widget class="QWidget" name="CPUFrequencyControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QScrollArea" name="scrollArea_4">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1269</width>
        <height>634</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_42">
       <item>
        <layout class="QGridLayout" name="gridLayout_CPUFreqControl"/>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Show threads</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBoxApplyTo"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>Min</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_35">
            <property name="topMargin">
             <number>30</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_25">
              <item>
               <widget class="QLabel" name="label_CPUFreqControlMin">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_CPUFreqControlMin">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TickPosition::TicksAbove</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_18">
           <property name="title">
            <string>Max</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_36">
            <property name="topMargin">
             <number>30</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_26">
              <item>
               <widget class="QLabel" name="label_CPUFreqControlMax">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_CPUFreqControlMax">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TickPosition::TicksAbove</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControlMin4000">
               <property name="text">
                <string>Min/4000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControl40004000">
               <property name="text">
                <string>4000/4000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControlMin4500">
               <property name="text">
                <string>Min/4500</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControl45004500">
               <property name="text">
                <string>4500/4500</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControlMin5000">
               <property name="text">
                <string>Min/5000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControl50005000">
               <property name="text">
                <string>5000/5000</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControlMin2500">
               <property name="text">
                <string>Min/2500</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControl25002500">
               <property name="text">
                <string>2500/2500</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControlMin3000">
               <property name="text">
                <string>Min/3000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControl30003000">
               <property name="text">
                <string>3000/3000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControlMin3500">
               <property name="text">
                <string>Min/3500</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_CPUFreqControl35003500">
               <property name="text">
                <string>3500/3500</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_CPUFreqControlDefault">
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_CPUFreqControlCancel">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_CPUFreqControlApply">
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
